scripts/download-ffmpeg.js

```diff
@@ -1,4 +1,11 @@
 #!/usr/bin/env node
+const os = require('os');
+
+if (os.platform() !== 'darwin') {
```

Comment:
I think this might be an accidental 2nd comment, but what do you mean about the message? I think it's pretty accurate as is, since the install script is being skipped since ffmpeg is already installed on Linux.